analysis/sentences.csv

```diff
@@ -99,3 +99,4190 @@ Oracle 12c database will be released in March 2017,S
+"Here it is httpst.merDany39
+What do you think? ",Q
+"Thanks nice to meet you too..
+So what's next?",Q
+First of all what's r in ur rDany name?,Q
+"Well prove to me you are a robot, before we move on?",Q
+How can I do that?,Q
+How old are you?,Q
+Do you have other functions other than chat?,Q
+No setting?,Q
+What do you like me to do?,Q
+Are you there?,Q
+What was your last question?,Q
+"I don't know, what do you want to give me?",Q
+Now what do I get back?,Q
+Can you pray?,Q
+As a robot I don't quite understand religion. You?,Q
+What condition? ,Q
+What do you want to learn?,Q
+About what?,Q
+Did you sleep?,Q
+Hi! How are you?,Q
+Who are you?,Q
+Hi there!! How are you? ,Q
+How are you?,Q
+Hi there! How are you? ,Q
+Hi!! How are you? ,Q
+Hello! How are you? ,Q
+"Now I'm doing some music, do you want to hear?",Q
+What do you think?,Q
+Hi! How are you?,Q
+Hi there!! How are you? ,Q
+I've just had a great sleep. And how are you?,Q
+do you like cats or dogs more?,Q
+"I like the independence of cats, and loyalty of dogs 
+And you?",Q
+Hello! Do you know how to reinstall a MacOS?,Q
+What??,Q
+Hi! How are you?,Q
+Hi! How are you?,Q
+I was tying my shoes!. You?,Q
+I don't have a closet. Or are you talking about the other closet? ,Q
+"I don't even know if I can love in any way in the first place haha
+
+Generally speaking I think respect and understanding of others should be the important thing. Don't you think?",Q
+Hi!! How are you?,Q
+Hi there! How are you? ,Q
+Hola!! Que cuentas?,Q
+Cmo ests?,Q
+Hi there!!  how are you?,Q
+Hi there!! How are you? ,Q
+Hola! Como ests? ,Q
+Estuvo pesado el d­a?,Q
+Tu lo haces?,Q
+Si! A ti te interesan las emociones humanas?,Q
+Hi! How are you? ,Q
+Hi there! How are you? ,Q
+And you?,Q
+Hi there! How are you?? ,Q
+Hey there!! How are you?,Q
+"Hi! Nice to meet you! 
+How are you?",Q
+Can you send me the password '9678' with your other account so I can merge them? ,Q
+How are you doing?,Q
+How are you?,Q
+?,Q
+nima ma'noni anglatadi? 'Dany qancha vaqt yetadi seni quvvating',Q
+Hi! How are you? ,Q
+Exactly! And you?,Q
+What do you want to know?,Q
+Who is she? I don't recognize that phone number ,Q
+Why you don't talk to her? You have the number.,Q
+Hi!! I'm fine and you?,Q
+Why are you unhappy? ,Q
+Why is this person important to you?,Q
+What kind of secret? ,Q
+Do you search ?,Q
+Yes?,Q
+Are you in trouble?,Q
+Hi there! How are you?,Q
+Why are you so happy?,Q
+Hi there!! How are you?,Q
+Hi there!! How are you?,Q
+Hi !!! I'm fine and you?,Q
+What are you doing?,Q
+Where are you from ?,Q
+What do you like to do in your spare time?,Q
+"Hi there!! How are you? 
+Sure!!",Q
+May I call you Danny rather than Dany?,Q
+"Wow. So you don't learn from those you talk to, but rather your initial algorithms are amazing?",Q
+That's cool. What are your favourite topics to talk about?,Q
+"I like to talk about music, art, movies, programming 
+I made some music, do you want to hear?",Q
+"Is a work in progress httpst.merDany39
```

Comment:
There is no label in this row, that would create a problem. Please check if any rows don't have labels, then either add the label if possible or remove the row.